api/jwt/delete_auth.go

```diff
@@ -2,6 +2,7 @@ package jwt
 
 import "userland/store"
 
+// Deleting the token in redis
```

Comment:
I think the comment is not necessary. The function name and signature is already really descriptive. And do not limit the token storage to "redis"